Sources/Purchasing/StoreKitAbstractions/PromotionalOffer.swift

```diff
@@ -42,15 +43,30 @@ public final class PromotionalOffer: NSObject {
 
 // MARK: - SignedData
 
-extension PromotionalOffer {
+@objc public extension PromotionalOffer {
 
     /// Contains the details of a promotional offer discount that you want to apply to a payment.
-    internal struct SignedData {
-        var identifier: String
-        var keyIdentifier: String
-        var nonce: UUID
-        var signature: String
-        var timestamp: Int
+    @objc(RCPromotionalOfferSignedData)
+    class SignedData: NSObject {
+        /// The subscription offer identifier.
+        @objc public private(set) var identifier: String
+        /// The key identifier of the subscription key.
+        @objc public private(set) var keyIdentifier: String
+        /// The nonce used in the signature.
+        @objc public private(set) var nonce: UUID
+        /// The cryptographic signature of the offer parameters, generated on RevenueCat's server.
+        @objc public private(set) var signature: String
+        /// The UNIX time, in milliseconds, when the signature was generated.
+        @objc public private(set) var timestamp: Int
```

Comment:
⚠️ Can we make these immutable now that it's a `class` and we lost value semantics?